[rocprofiler-systems] Add Fortran main detection to rocprof-sys-instrument to avoid instrumenting around C "main" wrapper (#1322)
ROCm/rocm-systems: projects/rocprofiler-systems/docs/how-to/instrumenting-rewriting-binary-application.rst

   patchelf --remove-rpath <exe-or-library>
   patchelf --set-rpath '/home/user' <exe-or-library>

Main function detection issues
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> If you encounter issues with ROCm Systems Profiler not properly detecting or instrumenting

Reviewer: ```suggestion
If ROCm Systems Profiler fails to detect or instrument your program's main function, you can explicitly specify it using the ``--main-function`` (or ``-m``) option.
```

> your program's main function, you can explicitly specify the main function using

Reviewer: ```suggestion
```

> the ``--main-function`` (or ``-m``) option.

Reviewer: ```suggestion
```
